extensions/mssql/package.nls.json

```diff
@@ -43,7 +43,8 @@
 	"mssql.exportSqlAsNotebook": "Export SQL as Notebook",
 	"mssql.configuration.title": "MSSQL configuration",
 	"mssql.query.displayBitAsNumber": "Should BIT columns be displayed as numbers (1 or 0)? If false, BIT columns will be displayed as 'true' or 'false'",
-	"mssql.query.maxXmlCharsToStore": "Number of XML characters to store after running a query",
+	"mssql.query.maxXmlCharsToStore": "Maximum number of XML characters to store after running a query. Default value: 2,097,152. Valid value range: 1 to 2,147,483,647.",
```

Comment:
Maybe specify even more `Maximum number of characters to store for each value in XML columns after running a query.` (and similarly `Maximum number of characters/bytes to store for each value in character/binary columns after running a query.`